eclipse-scout-core/src/form/fields/groupbox/GroupBoxLayout.ts

```diff
@@ -1,19 +1,19 @@
 /*
- * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
  *
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-import {AbstractLayout, Dimension, EventHandler, FormField, graphics, GroupBox, HtmlComponent, HtmlCompPrefSizeOptions, HtmlEnvironment, MenuBarLayout, PropertyChangeEvent, ResponsiveManager, scrollbars} from '../../../index';
+import {AbstractLayout, Dimension, EventHandler, FormField, graphics, GroupBox, HtmlComponent, HtmlCompPrefSizeOptions, HtmlEnvironment, MenuBarLayout, PropertyChangeEvent, ResponsiveManager, scout, scrollbars} from '../../../index';
 import $ from 'jquery';
 
 export class GroupBoxLayout extends AbstractLayout {
   groupBox: GroupBox;
   protected _htmlPropertyChangeHandler: EventHandler<PropertyChangeEvent<any, HtmlEnvironment>>;
-  protected _statusWidth: number;
+  protected fieldStatusWidth: number;
```

Comment:
This is an internal property, please use `_`-prefix.

Reply:
Changed to _fieldStatusWidth